tests/foreman/cli/test_remoteexecution.py

```diff
@@ -936,7 +936,7 @@ def test_positive_install_ansible_collection(self, fixture_sca_vmsetup, module_g
 class TestRexUsers:
     """Tests related to remote execution users"""
 
-    @pytest.fixture(scope='module')
+    @pytest.fixture(scope='class')
```

Comment:
Since we are moving towards ReX and Ansible as defaults in future releases to replace this fixtures would be really helpful to move them to `pytest_fixtures` package under REX fixture module. Probably there you need not to change the scope to class level from existing module! But if you still want to continue here just rename the fixtures to prepend with `class_` not `module_`

Not blocking your PR fo this though!

Reply:
updated, thanks!